src/components/WatchVideo.vue

```diff
@@ -152,6 +152,7 @@
                 v-if="showDesc && sponsors && sponsors.segments"
                 v-text="`${$t('video.sponsor_segments')}: ${sponsors.segments.length}`"
             />
+            <div v-if="showDesc && video.category" v-text="`${$t('video.category')}: ${video.category}`" />
```

Comment:
Everything that is only visible when `showDesc` is true should bee wrapped into a `template` to avoid having the same if condition multiple times in a row.

Reply:
:+1: